triveni-codev/GBWebApp-dev
# Darlp networks link caption settings 1203216026809192

Hi @triveni-codev @chriscodev  @pauljustinhernandez 

Kindly check this PR. This is only for Linked Schedule Caption Settings tab under Services only for ATSC Terrestrial type

ASANA: https://app.asana.com/0/1201943683850147/1203216026809192


To test:

-should have the same set and order of languages from Java App
-should be able to retrieve linked schedule caption settings from Java App and add/update them
-should be able to add one or more linked schedule caption settings
-should be able to update one or more linked schedule caption settings
-cancellation should be working fine including all upper modals: networks, channels, services and service caption settings modal
-test Big Commit for both Adding and Updating
-you may test a combination of both default service caption settings and linked schedule caption settings
